spk/demoservice/Makefile

```diff
@@ -38,7 +38,7 @@ PRE_COPY_TARGET = demoservice_pre_copy
 
 include ../../mk/spksrc.spk.mk
 
-.PHONY: demoservice_install
+.PHONY: demoservice_pre_copy
```

Comment:
Good catch.  While at it, might be good creating a demodocker service?

Reply:
Is it working for you?